pkg/operator/ceph/cluster/controller.go

```diff
@@ -176,6 +177,24 @@ func (c *ClusterController) StartWatch(namespace string, stopCh chan struct{}) {
 	} else {
 		logger.Infof("Disabling hotplug orchestration via %s", disableHotplugEnv)
 	}
+
+	// watch for "rook-ceph-operator-config" ConfigMap
+	configMapName := "rook-ceph-operator-config"
+	_, rookCephOperatorConfigController := cache.NewInformer(cache.NewFilteredListWatchFromClient(c.context.Clientset.CoreV1().RESTClient(),
+		"configmaps", operatorNamespace, func(options *metav1.ListOptions) {
+			metav1.SingleObject(metav1.ObjectMeta{
+				Name:      configMapName,
+				Namespace: operatorNamespace,
+			})
+		}), &v1.ConfigMap{},
+		0,
+		cache.ResourceEventHandlerFuncs{
+			AddFunc:    c.operatorConfigAdd,
+			UpdateFunc: c.operatorConfigUpdate,
+			DeleteFunc: nil,
+		})
+
+	go rookCephOperatorConfigController.Run(stopCh)
```

Comment:
Logs is flooded with
```bash
E0226 11:54:32.194117       7 streamwatcher.go:109] Unable to decode an event from the watch stream: unable to decode to metav1.Event
W0226 11:54:32.194297       7 reflector.go:289] github.com/rook/rook/pkg/operator/ceph/cluster/controller.go:197: watch of *v1.ConfigMap ended with: very short watch: github.com/rook/rook/pkg/operator/ceph/cluster/controller.go:197: Unexpected watch close - watch lasted less than a second and no items received
```
Seems like increasing the resync period doesn't fix this.

Reply:
But, the feature works just fine.

Reply:
How often does that entry get logged? Did it just start happening? Perhaps it's related to watching the single object instead of based on the label. I'd like to not flood the operator logs.

Reply:
> How often does that entry get logged? 
>
Like every second.
> Did it just start happening?
>
Yes.
> Perhaps it's related to watching the single object instead of based on the label. I'd like to not flood the operator logs.
>
Yes, looking at what can be done. Looks like even Go tools can't figure out what's wrong in these blocks of code (even syntax error was uncaught).

Reply:
Fixed it.